ShareExtension/ShareExtension/Components/PhotoPickerView.swift

```diff
@@ -92,12 +62,8 @@ private extension PhotoPickerView {
         static let message = "There was an issue with uploading your image. Please try again."
         static let text = "OK"
     }
-
-    enum Share {
-        static let caption = "Share this image."
-    }
 }
 
 #Preview {
-    PhotoPickerView()
+    PhotoPickerView(images: Binding.constant([IdentifiableImage(image: Image("rabbit"))]))
```

Comment:
Same for the images, if more static images will be added to the application, please extract it like for the strings and numbers.